README.md

```diff
@@ -58,6 +58,98 @@ running Go 1.9 or higher.
 
 - [Getting Started with Azure Blob Service in Go](https://github.com/Azure-Samples/storage-blob-go-getting-started)
 
+# Generate code
+
+## Generate the SDK
+
+1. [Install AutoRest](https://github.com/Azure/autorest#installing-autorest).
+
+### The easy way
+
+This repo contains a tool to generate the SDK, which depends on the golang tags from the readme files in the Azure REST API specs repo. The tool assumes you have an [Azure REST API specs](https://github.com/Azure/azure-rest-api-specs) clone. This will generate a single service, including all its API versions.
+
+1. `tools/generator`
+
+1. `go install`
+
+1. Add `GOPATH/bin` to your `PATH`, in case it was not already there.
+
+1. Call the generator tool like this...
+
+``` cmd
+generator –r –v –l=logs/output/folder –version=version path/to/your/swagger/repo/clone/specification/service
+```
+
+ For example
+
+``` cmd
+generator –r –v –l=temp –version=v11.2.0-beta C:/azure-rest-api-specs/specification/network
+```
+
+### Generate the complete SDK
+
+All services, all API versions.
+
+1. Call the generator tool like this...
+
+``` cmd
+generator –r –v –l=logs/output/folder –version=version path/to/your/swagger/repo/clone
+```
+
+ For example
+
+``` cmd
+generator –r –v –l=temp –version=v11.2.0-beta C:/azure-rest-api-specs
```

Comment:
A lot of this redundancy could fall away if we refactored as per my comment near the top about sections.